docs/user_guide.rst

```diff
@@ -1,2 +1,44 @@
 User Guide
 ==========
+
+.. contents:: :depth: 3
+
+:Note: FoxPuppet uses Selenium as it's main driver. That means Selenium commands work
+       right along with FoxPuppet specific commands.
+
+Example
+-------
+
+This is an example for setting up FoxPuppet and running a simple check to see if the browser window is private::
+
+    from foxpuppet import FoxPuppet
+    from selenium.webdriver import Firefox
+
+
+    selenium = Firefox
```

Comment:
This should be `selenium = Firefox()`. Make sure you can copy/paste your examples into a Python interpreter.